### ISSUE TITLE
Active side bar item actions have poor contrast with Light+ V2

### ISSUE THREAD

**Issue by nickmccurdy** [microsoft/vscode#173238]
Type: <b>Bug</b>

Steps to Reproduce:

1. Use Light+ V2 color theme (`Preferences: Color Theme`)
2. Open a side bar view (i.e. Source Control)
3. Select an item in the side bar to make it active (i.e. a changed file)
4. Keep VSCode open in the foreground
5. Observe the constrast between the item action icons (white) and active item backgorund (light gray)

![VSCode screen shot](https://user-images.githubusercontent.com/927220/216509823-bc7ae1e0-c587-4fcd-a2e5-027caefed033.png)

VS Code version: Code 1.75.0 (e2816fe719a4026ffa1ee0189dc89bdfdbafb164, 2023-02-01T15:24:42.903Z)
OS version: Darwin x64 21.6.0
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz (4 x 2700)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|12, 10, 6|
|Memory (System)|8.00GB (0.12GB free)|
|Process Argv|--crash-reporter-id 858be5e5-b2d4-4f5e-a561-5403bbbc360c|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
gitstash|art|5.1.0
astro-vscode|ast|0.28.2
format-on-auto-save|BDS|1.0.4
vscode-tailwindcss|bra|0.9.7
vscode-eslint|dba|2.2.6
prettier-vscode|esb|9.10.4
copilot|Git|1.71.8269
copilot-labs|Git|0.9.668
quicktype|qui|12.0.46
vscode-yaml|red|1.11.0
code-spell-checker|str|2.16.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaac:30438847
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30577860
nodejswelcome1:30587005
3biah626:30602489
pyind779:30657576
f6dab269:30613381
pythonsymbol12cf:30657549
vscup:30628656

```

</details>

**Comment by eamodio** [comment]
Same in quickpicks
![image](https://user-images.githubusercontent.com/641685/216832462-75c7c675-9bde-448d-b1a6-d85f96be958f.png)

**Comment by daviddossett** [comment]
Known bug that will be fixed by #172668

**Comment by daviddossett** [comment]
Fixed via #172668 👍
